pkg/secrethub/http.go

```diff
@@ -121,6 +123,13 @@ func newHTTPClient(authenticator auth.Authenticator, opts *ClientOptions) *httpC
 	}
 }
 
+func (c *httpClient) Authenticate(in interface{}) (*api.Session, error) {
```

Comment:
As said before, this is far more a `CreateSession` than `Authenticate` which is an ambiguous word to me.